src/app/shared/services/dialog.service.ts

```diff
@@ -33,4 +33,20 @@ export class DialogService {
       onOk();
     });
   }
+
+  public openActivationDialog(nameOfRundownToActivate: string, onOk: () => void): void {
```

Comment:
This isn't an optimal way of using the DialogService, but we talked about this already, so just leaving the comment as a reminder

Reply:
Changed to fit what we talked about!

Reply:
B-e-a-utiful